src/main/java/zendo/games/zenlib/ZenMain.java

```diff
@@ -138,4 +142,12 @@ public void render() {
         screen.render(zenAssets.batch);
     }
 
+    private void loadVisUI() {
+        if (config.uiSkinPath == null) {
+            VisUI.load(VisUI.SkinScale.X2);
```

Comment:
Consider adding a log message here like `No uiskin specified in config, loading default VisUI skin (x2 scale)`.